sos/plugins/redis.py

```diff
@@ -46,26 +61,18 @@ def setup(self):
             ])
 
     def postproc(self):
-        self.do_file_sub(
-            "/etc/redis.conf",
-            r"(masterauth\s).*",
-            r"\1********"
-        )
-        self.do_file_sub(
-            "/etc/redis.conf",
-            r"(requirepass\s).*",
-            r"\1********"
-        )
-        self.do_path_regex_sub(
-            self.var_puppet_gen + "/etc/redis.conf*",
-            r"(masterauth\s).*",
-            r"\1*********"
-        )
-        self.do_path_regex_sub(
-            self.var_puppet_gen + "/etc/redis.conf*",
-            r"(requirepass\s).*",
-            r"\1*********"
-        )
-
-
+        for path in ["/etc/",
+                     self.var_puppet_gen + "/etc/",
+                     "/etc/opt/rh/rh-redis32/",
+                     "/etc/opt/rh/rh-redis5"]:
```

Comment:
Maybe better to parametrise the SCL path via `add_copy_spec_scl` method from https://github.com/sosreport/sos/blob/master/sos/report/plugins/__init__.py#L1946, if we utilize SCLPlugin already?